KPP/fullchem/gckpp_Rates.F90

```diff
@@ -619,7 +619,7 @@ SUBROUTINE Update_RCONST ( )
   RCONST(186) = (2.40d-10)
   RCONST(187) = (GCARR_ac(2.10d-11,-2200.0d0))
   RCONST(188) = (GCARR_ac(7.20d-14,-1070.0d0))
-  RCONST(189) = (GCJPLAC_ababac(3.4e-31,1.6d0,2.3d-11,0.2d0,5.3d-12,2.0d2,0.6d0))
+  RCONST(189) = (GCJPLAC_ababac(3.4d-31,1.6d0,2.3d-11,0.2d0,5.3d-12,2.0d2,0.6d0))
   RCONST(190) = (1.30d-11)
```

Comment:
As mentioned above, better to change this entry in the fullchem.eqn file and then rebuild the mechanism.

Reply:
Also would need to update this in both `KPP/fullchem/fullchem.eqn` and `KPP/custom/custom.eqn`